### PR TITLE
Doing exercises from Microverse dashboard

### PR DESCRIPTION
**Pull request by Jessemccallums**
Requirements

I Created a new GitHub repo.
I Created a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
I added a .gitignore file.
I Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.
I added a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
I added 3 files (each one in a separate commit):
A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
A .css file that includes one class with styles for your header (make it in your favorite color).
A test.md file with the text "This file should be ignored by git".
And I Opened a pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,22 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# hellomicroverse
 
-> Description the project.
+> This is a project from Microvrse exercises.
```
Comment (tresorsawasawa) [inline README.md]: - [ ] Good job :+1: by adding a description of your project. So, I noticed that it is not more descriptive according to the project you completed. I suggest you stick an example from the one in the project's requirements to make your documentation more professional.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+body {
+  background-color: grey;
+}
+
+h1 {
+  color: aliceblue;
+}
```
Comment (tresorsawasawa) [inline main.css]: - [ ] Good job :+1: so far by adding styling to your project. So, I noticed from [the project's requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20.css%20file%20that%20includes%20one%20class%20with%20styles%20for%20your%20header%20(make%20it%20in%20your%20favorite%20color).) that you should have one **class** you use to style. I suggest you add one class to your` index.html` file and use it in styling.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
+}
```
Comment (tresorsawasawa) [inline .vscode/settings.json:3]: - [x] I think it would be better :100:  to add every `file` and `folder` that is not related to the project in `.gitiggnore` to make your project clean and for best practices. As for this, you can also add it in **.gitignore** and remove it from GitHub.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css"  type="text/css"  />
+    <title>Document</title>
```
Comment (tresorsawasawa) [inline index.html:8]: - [ **OPTIONAL** ] I think it would be better to assign to your project a meaningful name. So, it can even give a short idea before any user accesses your page. I  suggest you change this to a meaningful one instead of `Document`.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @Jessemccallums,
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 

## Required Changes :recycle: :recycle: 

- [x] Kindly add a description to your pull request to make it more professional. You should tell what changes are inside the project. I suggest you stick to the project's requirements as they are tasks you completed in this project.
- [x] Good job so far by using the Micriverse README template. So, I noticed that you miss some important sections in your README that are crucial to have in a  project's documentation. I suggest you add the **Getting Started** section, so you can add like:
 - Prerequisites: you can describe some tools needed before cloning the project e.g: Git, IDE(Vscode, Sublime,...)
- Setup: Add some commands to clone the project, access the cloned directory, and/or even how to run/open it locally in the browser.
- Install: command to install packages dependencies after cloning. `npm install`
- [ ] **Check the comments under the review**.

## Optional suggestions
- [ **OPTIONAL** ] Good job so far by giving a meaningful full name to your feature branch, but it can be good if you rename it to a project similar name.
- [ **OPTIONAL** ] I also suggest you improve your Pull Request title to the project title similar, I think this will look better than they used.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — APPROVED** [review]
Hi @Jessemccallums , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

- [ OPTIONAL] as previous reviewer said, You are recommend to use a good project title, just improve this `<title> Document</title>`

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
